### ISSUE TITLE
Documentation updates

### ISSUE THREAD

**Issue by justincely** [spacetelescope/stak#67]
- User doc for Hselect
- link to http://stak.userecho.com/
- get automodapi from pip instead of installing from source (rtd-pip-requirements)
- page for command-line functionality
- linking in notebooks
- summarize package in notes section

**Comment by SaraOgaz** [comment]
@justincely  Ok, so I messed around with linking one notebook from another and it's a bit tricky.  So far I've seen three options

a) you can link to a sphinx header in another doc file, but we're using the wrong header type (not sphinx syntax but markdown) It also renders the tagging in the cell in the notebook, so it doesn't look nice.

b) you can use rst notation to link to another html file, but this involves knowing the html ID that's going to get assigned to the final doc files, sphinx does not recommend doing this.

c) We can link a notebook to another notebook, so at least we could get linkage when someone was using the notebooks as actual notebooks, however, this then acts as a broken link when you are looking at the doc page version of the notebook....

Basically, in my attempts so far I haven't found a nice solution.  Maybe we can find and ask a sphinx expert?

I've kept track of which notebooks have links to _other_ notebooks.  For links within the same notebook, those are all done.

**Comment by SaraOgaz** [comment]
@justincely   Are we STScI Astronomy Kit, or Analysis Toolkit?  I vote Astronomy Kit since we're focusing in on astronomy related stuff.

**Comment by SaraOgaz** [comment]
@justincely,

I'm putting in a PR with the main doc updates so I can start getting comments on it, in the meantime I have some questions about the remaining items

- I'm not sure how/what you wanted to do with changing our installation to pip, do we just need to do that in the .travis.yml file, or do we need to add a file for rtd specifically?
- We still need to iterate on the linking in the notebook issue (from my above comments)
- I don't remember what we meant by "summarize package in notes section"
- Also, we should resolve the "official package name thing", then I can check it's consistent across all our doc stuff

**Comment by justincely** [comment]
- Space Telescope Astronomy Kit
- automodapi is good - we get it from pip right now, must have already changed it.
- does the linking behavior you found change with @pllim's conversion procedure?  i.e. are the behaviors you found a consequence of the conversion or just the final documents themselves?
- Summarizing the package means, in the notes section at the top, give a paragraph or two that gives a high-level summary of what is contained in the package.

**Comment by SaraOgaz** [comment]
Closing this issue, pretty much all these parts are finished, or irrelevant due to changes in the package.
